src/server/services/buurt/meldingen-subcategories.ts

```diff
@@ -0,0 +1,630 @@
+export function transformSlugToCategorie(slug: string | undefined) {
+  return meldingenSubCategories.find((c) => c.id === slug)?.name;
+}
+
+export const meldingenSubCategories = [

```

Comment:
Je zou ook default iets kunnen doen als:

```
let name = meldingenSubCategories.find((c) => c.id === slug)?.name;

if (!name) {
  name = capitalizeFirstLetter(slug.replace(/\-/g, ' '))
}

```
Dan hoef je alleen de uitzonderingen in de code op te nemen.